content/en/docs/Tasks/make-operator-part-of-catalog.md

```diff
@@ -19,8 +19,8 @@ COPY manifests manifests
 RUN ./bin/initializer -o ./bundles.db
 
 FROM scratch
-COPY --from=builder /build/bundles.db /bundles.db
-COPY --from=builder /build/bin/registry-server /registry-server
+COPY --from=builder /bundles.db /bundles.db
+COPY --from=builder /bin/registry-server /registry-server
```

Comment:
It looks like you need to rebase against master as you're picking up a change that was recently merged.